packages/tokens/properties/color/base.json

```diff
@@ -60,6 +60,7 @@
   },
   "eds": {
     "color": {
+      "transparent": { "value": "transparent" },
```

Comment:
I needed this in tailwind, but I should probably just add it to a config - no reason to make a variable for it.